avocado/utils/ssh.py

```diff
@@ -214,3 +214,28 @@ def quit(self):
         :rtype: bool
         """
         return self._master_command('exit')
+
+    def copy_files(self, source, destination, recursive=False):
+        """
+        Copy Files to and from remote through scp session
+        :param source: Local file
+        :type: str
+        :param destination: Remote file location
+        :type: str
+        :param recursive: Scp option for copy file. if set to True
+                          copy files inside directory recursively.
+        :type: boot
+        :returns: Whether the files copy successfully or not.
+        :rtype: bool
+        """
+        cmd = path_utils.find_command('scp')
```

Comment:
Since `find_command()` can raise a `CmdNotFoundError` exception you could try to capture that exception. Maybe add the find_command() line inside the try.. except to avoid having two excepts...

Reply:
@beraldoleal but better to have in process.run() too.

Reply:
> @beraldoleal but if i handle in one try ..except for two others lines also come inside it so better to have separate.